en/docs/guides/identity-lifecycles/role-based-permissions.md

```diff
@@ -686,7 +664,21 @@ xmlns:ser="http://service.ws.um.carbon.wso2.org"&gt;
 
 -->
 
+## Configure login permissions for a user
```

Comment:
the purpose of this section is not clear.
The title is `Configure login permissions`. However, what we seem to be doing is assign the `admin` role to a user. Are we trying to say that to get `login` permissions, the `admin` role is required?

I assume this is what we are trying to address here. Also, this section can be right at the top of the page, that way you can point off to the `Access permissions` section, which describes the login permissions in detail.

**Assign permissions to a user**

Permissions are enabled for a user role. Therefore, you need to first have roles created in your tenant with the required permissions. You can then assign the role to users. Users with the `admin` role are granted all permissions by default. 

As an admin user, if you want to  update the permissions of a user, follow the steps given below.

<steps on how to enable permissions for a user role. Can point to the doc on creating roles and proceed to this page.>
<steps on how to assign a role to a user>